tests/aws/services/stepfunctions/v2/services/test_ecs_task_service.py

```diff
@@ -0,0 +1,516 @@
+import json
+
+import aws_cdk as cdk
+import aws_cdk.aws_ecs as ecs
+import aws_cdk.aws_stepfunctions as sfn
+import aws_cdk.aws_stepfunctions_tasks as tasks
+import pytest
+from localstack_snapshot.snapshots.transformer import RegexTransformer
+
+from localstack.testing.pytest import markers
+from localstack.utils.analytics.metadata import is_license_activated
+from tests.aws.services.stepfunctions.utils import launch_and_record_execution
+
+
+# TODO: figure out a better way, maybe via marker? e.g. @markers.localstack.ext
+@pytest.mark.skipif(condition=not is_license_activated(), reason="integration test with pro")
+class TestTaskServiceECS:
```

Comment:
The file shows an interesting problem emerging: handling multiple infra configurations. Just highlighting it since this is something I'd like to investigate further in the future. Might make sense to use this test class as a sample.